shotover-proxy/tests/helpers/mod.rs

```diff
@@ -145,12 +145,19 @@ impl ShotoverManager {
     }
 
     #[allow(unused)]
-    pub fn cassandra_connection(&self, contact_points: &str, port: u16) -> Session {
+    pub fn cassandra_connection(
+        &self,
+        contact_points: &str,
+        port: u16,
+        username: &str,
+        password: &str,
```

Comment:
Can we just use hardcode the username/password instead of making it an arg?
Or do you plan to use a custom one somewhere?

Reply:
Nope, good idea.